> Solving the next problem at: 2024-09-27 08:36:01 UTC
> Search algorithm: bfs
> Instance ID=miconic/ps2-3-err-rate-0-5
> Plan length=7
> Domain class: miconic
> Instance name: ps2-3-err-rate-0-5
> Ground plan:
['(up f0 f1)', '(board f1 p0)', '(up f1 f2)', '(board f2 p1)', '(up f2 f3)', '(depart f3 p1)', '(depart f3 p0)']
> Lifted plan:
['up', 'board', 'up', 'board', 'up', 'depart', 'depart']
>>>  Vanilla ground repair length:
1

>>>  Vanilla ground repair:
up,(= ?f1 ?f1),precPos,-1

An error occurred during A* search: name 'logger' is not defined
Traceback (most recent call last):
  File "/home/nader/projects/lifted-white-plan-domain-repair/instance_solver.py", line 70, in solve_instance
    path, goal_node = searcher.find_path(log_interval=log_interval)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nader/projects/lifted-white-plan-domain-repair/search_partial_grounding/search_algorithms.py", line 49, in find_path
    for neighbor in current_node.get_neighbors():
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nader/projects/lifted-white-plan-domain-repair/search_partial_grounding/node.py", line 167, in get_neighbors
    logger.debug(f'> Current state:\n{self.current_state}')
    ^^^^^^
NameError: name 'logger' is not defined. Did you mean: 'self.logger'?

> Time spent on this problem: 0.0 hours = 0.0 minutes = 0.22 seconds
> Subprocess finished.
